indexer-app/src/cmr/indexer/data/index_set.clj

```diff
@@ -662,6 +662,8 @@
    :provider-id.lowercase (m/doc-values m/string-field-mapping)
    :variable-name (-> m/string-field-mapping m/stored m/doc-values)
    :variable-name.lowercase (m/doc-values m/string-field-mapping)
+   :variable-alias (-> m/string-field-mapping m/stored m/doc-values)
```

Comment:
It doesn't have to be stored unless you return it in a search response. If you think there's a chance it will eventually be returned in either the JSON or XML references formats for variables (alias is a commonly needed variables field), then I'm fine with going ahead and storing it now rather than ending up with two fields because we have to change it later on to store it.

Reply:
We've decided to keep it as stored since it's an important field.